main/common/aperture.js

```diff
@@ -163,6 +165,8 @@ const stopRecording = async () => {
     return;
   }
 
+  await callPlugins('willStopRecording');
```

Comment:
Thinking of removing this as it might cause recordings to continue longer than the user clicked. If not removing at least not `await`ing

Reply:
Agreed, should be removed.